tests/unit/policy_test.py

```diff
@@ -894,109 +894,80 @@ def test_surface_policy_moves_back_to_object(self):
+                #  22   | MoveTangentially | True        | False          | dynamic_call
                 # falls off object
-                #  46   | OrientHorizontal | True        | False          | touch_object
-                #  47   | OrientHorizontal | True        | False          | touch_object
-                #  48   | OrientHorizontal | True        | False          | touch_object
-                #  49   | OrientHorizontal | True        | False          | touch_object
-                #  50   | OrientHorizontal | True        | False          | touch_object
-                #  51   | OrientHorizontal | True        | False          | touch_object
-                #  52   | OrientHorizontal | True        | False          | touch_object
-                #  53   | OrientHorizontal | True        | False          | touch_object
-                #  54   | OrientHorizontal | True        | False          | touch_object
-                #  55   | OrientHorizontal | True        | False          | touch_object
-                #  56   | OrientHorizontal | True        | False          | touch_object
-                #  57   | OrientHorizontal | True        | False          | touch_object
-                #  58   | OrientVertical   | True        | False          | touch_object
+                # dataloder runs TouchObject positioning procedure in a tight loop only
+                # exposing a new observation at the end of the loop
+                #  23   | MoveForward      | True        | False          | touch_object
                 # back on object
-                #  59   | OrientHorizontal | True        | False          | dynamic_call
-                #  60   | OrientVertical   | False       | True           | dynamic_call
-                #  61   | MoveTangentially | True        | False          | dynamic_call
+                #  24   | MoveForward      | True        | False          | dynamic_call
+                #  25   | OrientHorizontal | True        | False          | dynamic_call
+                #  26   | OrientVertical   | KeyError    | True           | dynamic_call
+                #  27   | MoveTangentially | True        | False          | dynamic_call
                 # falls off object
-                #  62   | OrientHorizontal | True        | False          | touch_object
-
-                # Motor-only touch_object steps
-                if (
-                    13 <= loader_step <= 25
-                    or 29 <= loader_step <= 42
-                    or 46 <= loader_step <= 58
-                    or loader_step == 62
-                ):
+                # dataloder runs TouchObject positioning procedure in a tight loop only
+                # exposing a new observation at the end of the loop
+                #  28   | MoveForward      | True        | False          | touch_object
+
+                # Motor-only last step of touch_object positioning procedure
+                if loader_step in [1, 13, 18, 23, 28]:
```

Comment:
These fixed numbers seem like they could be brittle. Where do they come from? How are they calculated? What steps would someone who needs to update this have to take to come up with new ones?